scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -233,13 +246,26 @@ def _stop_shared_object_managers(self):
         # also stop all shared object managers.
         for worker_id, memory_manager in self.__shared_object_managers.items():
             try:
-                memory_manager.shutdown()
+                memory_manager.shutdown_and_wait()
```

Comment:
Can we pass a parameter here to specify the wait time?  If not, at least add a comment to say how long the wait time is.

Reply:
It has a default keyword argument `timeout` which is 5 seconds for now.